http/pprof/service.go

```diff
@@ -0,0 +1,26 @@
+package pprof
```

Comment:
Why can't we use the same http server for pprof as used for WebSocket/REST?

Reply:
We can. I was just following how is it right now. I haven't made changes here. We can use the same HTTP server as long as the path is not conflicting.

Reply:
Actually, there is a bug in the current implementation. We haven't defined the paths explicitly, which is a requirement when using gorilla.